### DISCUSSION TITLE
the repo is static Dev ya sure

### DISCUSSION THREAD

**Discussion by TomTravis** [dnschneid/crouton#4730] [Polls]
who is the new fork if this is to remain static

![HP+49$](https://user-images.githubusercontent.com/46386261/167467687-195a1022-67eb-459e-bc40-60cea2aecabb.jpg)

**Comment by TomTravis** [comment]
![Screenshot at 2022-05-09 02-29-00](https://user-images.githubusercontent.com/46386261/167496536-17cc496d-ea6a-48b4-a04a-783d555687f9.png)

**Comment by TomTravis** [comment]
who wishes to test> the commit have been made on my fork so have fun seem to be stable

**Comment by TomTravis** [comment]
I have started some testing the Jammy Release Installed on my arm64 chromebook with Developer mode Crostini  available.
The x64 edu chrome book fail to open the deb files for install zstd seem to be  installed
this is the error for jammy, xenial installs no problem CHOSENEXTRACTOR  is not available 
halts the install
Permission are set all mount are exec not issues
crouton-master/crouton.build/debootstrap the error points to debootstrap in ../var..... log file

case $SUITE in
	 gutsy|hardy|intrepid|jaunty|karmic|lucid|maverick|natty|oneiric|precise|quantal|raring|saucy|trusty|utopic|vivid|wily|xenial|jammy|yakkety|zesty|artful|bionic|cosmic|disco|eoan|focal|groovy|hirsute) ;;
	*)
		# impish+ will use zstd compression, check if supported
		dpkg_zstd="$(dpkg-deb --help 2>/dev/null | grep ' zstd,' || :)"
		if [ -z "$EXTRACTOR_OVERRIDE" ] && [ -z "$dpkg_zstd" ]; then
			info CHOSENEXTRACTOR "%s uses zstd compression, setting --extractor=ar option" "$SUITE"
			export EXTRACTOR_OVERRIDE=ar
		fi
	;;
esac

**Comment by TomTravis** [comment]
This the update mate desktop targets with Jammy and xiwi runs super nice, some of the targets need to be updated

the crouton image is on a google drive
https://drive.google.com/file/d/1_rfoNvtl8CaT8lDeB33dGj1K901Cc7dc/view?usp=sharing Crouton_jammy

I did -r jammy -t mate, audio, xiwi,touch
 x11 xiwi starts up just fine

I have not found a web broswer yet

this is one a arm64 lenovo with crostini enabled

My older lenovo amd64 will not install jammy  stop after stage 1 extractor fails debootstrap fails Im sure this can be address i have zstd 
![Screenshot at 2022-05-11 13-57-52](https://user-images.githubusercontent.com/46386261/167971660-a423ee00-720b-41f1-a860-e991fccdf848.png)
installed 

have fun

**Comment by BKFCAW** [comment]
I'm missing something.  When I download and make crouton_jammy executable, I get a message "Refusing to exec...from noexec mount."  Maybe I downloaded incorrectly.

I'd like to try this out, because all of my attempts to install jammy with the official crouton version have failed at launching X11.  My focal chroot with xfce and mate both load without any trouble.

**Comment by CroutonIsFun** [comment]
The existing crouton is in a holding pattern so as to support EOL devices. David said the testing labs are no longer available, so you can't just go making changes willy nilly to the existing repository. I believe this is the proper approach.

You don't need anyone's permission to fork crouton and make it work for modern Chromebooks. Currently, sudo is not available in the crosh shell, so the "standard" work around is to enter the chroot in VT-2, start SSH, then go to the crosh shell and SSH into localhost, and finally, kick off a desktop with some command, like sudo startxfce4.

A more experimental approach is to install Chromebrew, then the crew_sudo package, which intercepts calls to sudo and makes it work once again in the crosh shell.

Another wrinkle is that the security folder has to be made writable, or you can't complete an installation and launch a chroot.

I've written a new how-to in the wiki, which shows the standard work around without the use of Chromebrew or other 3rd party tools:

[Community Supported Instructions](https://github.com/dnschneid/crouton/wiki/Updated-Installation-Instructions-for-Crouton)
